modules/contentbox/modules/contentbox-admin/views/contentStore/editorSelectorHelper.cfm

```diff
@@ -35,8 +35,10 @@ function pagerLink(page){
 	} );
 }
 function insertContentStore(slug){
-	var customContent = "{{{ContentStore slug='"+slug+"'}}}";
-	insertEditorContent( '#rc.editorName#', customContent );
+	var widgetContent = new CKEDITOR.dom.element( 'contentstore' );
```

Comment:
Unfortunately, this bleeds the implementtion of the editor used.  As you can see, this file has no concept of ckeditor or any other editor. Meaning that each editor module provides methods so you can interact with it, so this file reamins editor agnostic.  This bleeds into direct ckeditor.  

Look in the ckeditor module to see the methods we provide

Reply:
Any updates on this @cfmitrah ?

Reply:
Any updates on this @cfmitrah ?